scripts/perf-test/ebs.yaml

```diff
@@ -43,8 +46,8 @@ Resources:
                 apt-get -y update
                 apt-get install git
                 cd /mnt
-                git clone --depth 1 --branch svcomp20 \
-                        https://github.com/sosy-lab/sv-benchmarks.git
+                git clone --depth 1 --branch main \
```

Comment:
Main may not be stable, is this intended (there are other branches that might be stable)?